app/models/submission.rb

```diff
@@ -0,0 +1,54 @@
+require 'securerandom'
+
+class Submission < ActiveRecord::Base
+  belongs_to :project
+  has_many :jobs, dependent: :destroy
+
+  store :job_attrs
+
+  attr_accessor :start_frame, :end_frame, :project_dir
+
+  def submit
+    parse_frames
```

Comment:
Does parse_frames exist to set `@start_frame` and `@end_frame` for use when rendering the erb?

If so, it would be preferable to just define methods to return `start_frame` and `end_frame` that will parse the frames attribute on the fly i.e.

```ruby
def start_frame
  frames.split('-').first
end
```

Reply:
Done.